hack/run-local.sh

```diff
@@ -14,5 +14,11 @@ echo "Image: ${IMAGE}"
 echo "Release version: ${RELEASE_VERSION}"
 echo "Namespace: ${NAMESPACE}"
 
-${DELVE:-} ./ingress-operator start --image "${IMAGE}" --release-version "${RELEASE_VERSION}" \
+if [[ -v "ENABLE_CANARY" ]]; then
```

Comment:
TIL:

> f.  test/[/[[ have a new -v variable unary operator, which returns success if
>    `variable' has been set.

https://tiswww.case.edu/php/chet/bash/CHANGES

`[[ -v ]]` is new in Bash 4.2, so this will break on macOS.